src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaSection.js

```diff
@@ -0,0 +1,30 @@
+//@flow
+import React from 'react';
+import {Datagrid, DatagridStore} from 'sulu-admin-bundle/containers';
+
+type Props = {
+    adapters: Array<string>,
+    datagridStore: DatagridStore,
+    onMediaClick: (mediaId: string | number) => void,
+};
+
+export default class MediaSection extends React.PureComponent<Props> {
```

Comment:
Same question as with the `CollectionSection`, what's the reason for extracting it?

Reply:
Imo splitting up the collection section and the media section makes sense and leads to cleaner code